src/fitnesse/wikitext/parser/Image.java

```diff
@@ -55,14 +56,51 @@ public Maybe<Symbol> parse(Symbol current, Parser parser) {
             return makeImageLink(current, link.getValue(), imageProperty);
         }
         else if (name.isType(SymbolType.Text) || name.isType(WikiWord.symbolType)) {
-            Symbol list = new Symbol(SymbolType.SymbolList).add(new Symbol(SymbolType.Text, name.getContent()));
+            String imageUrl = parseImageUrl(parser);
+            Symbol list = new Symbol(SymbolType.SymbolList).add(new Symbol(SymbolType.Text, imageUrl));
             Symbol link = new Symbol(Link.symbolType).add(list);
             addOptions(link, options);
             return makeImageLink(current, link, imageProperty);
         }
         else return Symbol.nothing;
     }
 
+    /**
+     * https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs
+     */
+    private String parseImageUrl(Parser parser) {
+        StringBuilder imageUrl = new StringBuilder();
```

Comment:
Maybe move this to just after the check whether the contents starts with `data`, so we don't even allocate it if it is not needed.

Reply:
done :-)